### DISCUSSION TITLE
Windows sub linux  issue

### DISCUSSION THREAD

**Discussion by asifsultan-THD** [pybamm-team/PyBaMM#1855] [General]
I have installed WSL and going according to your instructions but this line is showing me error after this line
git clone https://github.com/pybamm-team/PyBaMM.git

Cloning into 'PyBaMM'...
fatal: unable to access 'https://github.com/pybamm-team/PyBaMM.git/': Could not resolve host: github.com

Please help out

**Comment by martinjrobins** [comment]
Hi @asifsultan-THD. I don't think this is an issue with PyBaMM. Looks like you don't have network access in your WSL environment, or cannot see your DNS.

**Reply by asifsultan-THD** [reply]
Thanks for pointingthe error I know then where the issue is. i just want to ask that I can use my windows OS for PyBamm Installation. It is only that  I would not be able to use optional dependency like scikit-odes. Actually I need Pybamm for getting simulated data for my machine learning training purpose. Will it be ok or I have to go through WSL only.
Thanks in Advance for your support

**Comment by martinjrobins** [comment]
PyBaMM can be installed both in Windows and WSL (some optional solvers are not available in windows), see windows instructions here: https://pybamm.readthedocs.io/en/latest/install/windows.html

**Comment by asifsultan-THD** [comment]
@martinjrobins Hello. Can we use PyBaMM to create simulated data and labelled data for Machine learning training purpose? Do you think it is a good idea?

**Comment by martinjrobins** [comment]
@tinosulzer is probably better able to answer this than me, see https://www.sciencedirect.com/science/article/pii/S2542435121002932
